scripts/copper/slab_with_na/prepare.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path, PosixPath
+
+from ase import Atom, Atoms
+
+from pseudopotential_calculator.calculations.slab import (
+    SlabOptimizationParams,
+    add_atom_onto_slab,
+    get_slab_calculator,
+    get_top_position,
+    repeat_slab,
+)
+from pseudopotential_calculator.castep import (
+    Castep,
+    CastepConfig,
+    load_calculator_atoms,
+)
+from pseudopotential_calculator.hpc import (
+    prepare_calculator_with_submit_script,
+    prepare_submit_all_script,
+)
+from pseudopotential_calculator.scripting import maybe_copy_files_to_hpc
+from pseudopotential_calculator.util import (
+    prepare_clean_directory,
+)
+
+SLAB_SIZE_PATH = Path("data/copper/slab/slab_size")
+FREE_LAYER_PATH = Path("data/copper/slab/free_layer")
+
+
+def _prepare_na_atom(slab_copper: Atoms) -> Atom:
+    na_rel_position = [0, 0, 1]
+    cu_position = get_top_position(slab_copper)
+    na_position = [x + y for x, y in zip(cu_position, na_rel_position)]
```

Comment:
x + y is misleadign here - it could be misread as x ccordinate plus y coordinate